text_formatter.go

```diff
@@ -217,9 +233,18 @@ func (f *TextFormatter) printColored(b *bytes.Buffer, entry *Entry, keys []strin
 	}
 
 	levelText := strings.ToUpper(entry.Level.String())
-	if !f.DisableLevelTruncation {
+	if !f.DisableLevelTruncation && !f.PadLevelText {
 		levelText = levelText[0:4]
 	}
```

Comment:
@richpoirier if you want to avoid multi-bool conditions, while keeping present behavior (someone is probably parsing the output and expecting it to remain consistent) here's a possibility:
```go
levelText = fmt.Sprintf(formatString, strings.ToUpper(entry.Level.String())
if !f.PadLevelText {
  levelText = strings.TrimSuffix(levelText, " ")
}
if !f.DisableLevelTruncation {
  levelText = levelText[0:4] // this also breaks for many non-english scripts btw
}
```

Reply:
I'll do this if it's still relevant after I hear back from either @richpoirier or @dgsb about this comment https://github.com/sirupsen/logrus/pull/977#issuecomment-506567331

Reply:
Alright! I'll change the PR to do this in a bit